templates/editSchedule.html

```diff
@@ -31,8 +31,8 @@
   </div>
 </template>
 
-<template name="unscheduledRound">
-  <div class="fc-event" data-round-id="{{ _id }}">{{ displayTitle }}</div>
+<template name="sortedRound">
+  <div class="fc-event {{ draggability isScheduled }}" data-round-id="{{ _id }}">{{ displayTitle }}</div>
```

Comment:
Just curious, is there a reason for passing this attribute (`isScheduled`) to the method instead of having the method look up the attribute itself?